helm-charts/chatqna/values.yaml

```diff
@@ -48,3 +48,11 @@ global:
   modelUseHostPath: ""
   # modelUseHostPath: /mnt/opea-models
   # modelUsePVC: model-volume
+
+  # Enabling HorizontalPodAutoscaler (HPA) will:
+  # - Overwrite existing PrometheusAdapter "adapter-config" configMap with ChatQnA specific custom metric queries
+  #   for embedding, reranking, tgi services
+  # Upstream default configMap:
+  #  - https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/deploy/manifests/config-map.yaml
+  horizontalPodAutoscaler:
```

Comment:
This will need "HorizontalPodAutoscaler (HPA) support" section in `chatqna` README too.   It can be same as in common components, with first clause updated e.g. to:
```
`horizontalPodAutoscaler` option enables HPA scaling for the TGI and TEI inferencing deployments:
```

Verification section can be omitted I think, it's enough to have it in TGI & TEI READMEs.

Options table entry description could be e.g:
```
HPA autoscaling for the TGI and TEI service deployments based on metrics they provide. See #pre-conditions and #gotchas before enabling!  (If one doesn't want one of them to be scaled, given service `maxReplicas` can be set to `1`)
```